Share/Share.Plugin.iOSUnified/ShareImplementation.cs

```diff
@@ -87,10 +93,18 @@ static ShareImplementation()
         /// <param name="text">Text to share</param>
         /// <param name="title">Title of the share popup on Android and Windows, email subject if sharing with mail apps</param>
         /// <param name="excludedActivityTypes">UIActivityType to exclude</param>
-        /// <returns>awaitable Task</returns>
-        public async Task Share(string text, string title = null, params NSString[] excludedActivityTypes)
+        /// <returns>True if the operation was successful, false otherwise</returns>
+        [Obsolete("Use Share(ShareMessage, ShareOptions)")]
+        public Task<bool> Share(string text, string title = null, params NSString[] excludedActivityTypes)
         {
-            await ShareInternal(title, text, null, excludedActivityTypes);
+            var shareMessage = new ShareMessage();
+            shareMessage.Title = title;
+            shareMessage.Text = text;
+
+            var shareOptions = new ShareOptions();
+            shareOptions.ExcludedUIActivityTypes = excludedActivityTypes?.Select(x => (string)x).ToArray();
```

Comment:
ExcludedUIActivityTypes can now be set on ShareOptions (using string instead of NSString for cross platform compatibility)